src/patternfly/components/Pagination/pagination.scss

```diff
@@ -0,0 +1,95 @@
+@import "../../patternfly-imports";
+
+.pf-c-pagination {
+  // Base
+  --pf-c-pagination--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-pagination--child--MarginRight: var(--pf-global--spacer--lg);
+
+  // Dropdown
+  --pf-c-pagination--c-dropdown__toggle--Color: var(--pf-global--Color--100);
```

Comment:
Is the component options-menu not dropdown? Maybe update the naming here

Reply:
Ah good catch, this isn't needed anymore either.